### ISSUE TITLE
Create v0 UI for management and stats analysis

### ISSUE THREAD

**Issue by relativityboy** [steemit/overseer#8]
Overseer needs a management and statistics analysis web UI. This is the meta-issue for that.

**Comment by jnordberg** [comment]
UI will depend on what backend we use. Currently we use rakam.io and they provide a (very basic) UI

**Comment by sneak** [comment]
No, this is internal-only.  It can be ugly as sin.  I'd be fine with `<table>`s typeset in Times as long as we can get a realtime view of the data.

**Comment by sneak** [comment]
v0 Requirements:

A funnel view that shows the trailing 24h and trailing 30d for the following numbers:

- how many click "sign up" on steem.io/steemit.com
- what percentage put in username and click next on signup form
- what percentage put in email and click next
- which percentage put in phone number and click next
- which percentage confirm phone
- which percentage get approved
- which percentage complete creation
- which percentage post a first post
- which percentage earn at least $1 within their first 30 days on site
- which percentage come back in the second month

This is our funnel.  It doesn't have to be pretty, but I want realtime stats on this information.

We can use rakam, or we can just jam the data into an SQL database - I don't much care right now.  I just want a URL I can visit that shows me these raw figures, and each as a percentage of the previous step's number.

**Comment by sneak** [comment]
v0 Requirements:

A funnel view that shows the trailing 24h and trailing 30d for the following numbers:

- how many click "sign up" on steem.io/steemit.com
- what percentage put in username and click next on signup form
- what percentage put in email and click next
- which percentage put in phone number and click next
- which percentage confirm phone
- which percentage get approved
- which percentage complete creation
- which percentage post a first post
- which percentage earn at least $1 within their first 30 days on site
- which percentage come back in the second month

This is our funnel.  It doesn't have to be pretty, but I want realtime stats on this information.

We can use rakam, or we can just jam the data into an SQL database - I don't much care right now.  I just want a URL I can visit that shows me these raw figures, and each as a percentage of the previous step's number.

This is the primary reason we redid the faucet, and this is the whole purpose (for the moment) that overseer exists.  (Future purposes might be achievements or other a/b testing analytics - but that is later.)

**Comment by sneak** [comment]
Plan to run something like `DELETE FROM events WHERE "event_date" < (CURRENT_DATE - INTERVAL '60 days');` periodically from cron (e.g. every night) so we aren't keeping user tracking data for longer than we need to analyze it.  I don't want us to get a subpoena a year from now and have a user's entire multi-month reading history to turn over; we don't need it for our own business purposes.
